sdk/modelsrepository/azure-iot-modelsrepository/README.md

```diff
@@ -0,0 +1,36 @@
+# Azure IoT Models Repository client library for Java
```

Comment:
This Readme will be filled out after the merge to master so I can use the links directly from master.